packages/ses/test/hermes-smoke.js

```diff
@@ -0,0 +1,53 @@
+// Hermes doesn't support native I/O,
+// so we concat the SES shim above,
+// when running this test on Hermes.
+
+/**
+ * Test calling SES lockdown.
+ */
+const testLockdown = () => {
+  lockdown();
+};
+
+/**
+ * TODO: Test creating a new Compartment.
+ */
+// eslint-disable-next-line no-unused-vars
+const testCompartment = () => {
+  // eslint-disable-next-line no-unused-vars
+  const c = new Compartment();
+};
+
+/**
+ * TODO: Test Compartment import hook and resolve hook.
+ */
+// eslint-disable-next-line no-unused-vars
+async function testCompartmentHooks() {
+  const resolveHook = a => a;
+
+  async function importHook() {
+    return {
+      imports: [],
+      exports: ['meaning'],
+      execute(exports) {
+        exports.meaning = 42;
+      },
+    };
+  }
+
+  const compartment = new Compartment({}, {}, { resolveHook, importHook });
+
+  const module = compartment.module('.');
+
+  const {
+    namespace: { _meaning },
+  } = await compartment.import('.');
+
+  assert(module);
+}
+
+testLockdown();
+
+// testCompartment();
+
+// testCompartmentHooks();
```

Comment:
Vestigial comments. Please expand or remove.

I look forward to running `test:hermes` under a future `hardened262` package to get better coverage.